SUMMARY.md

```diff
@@ -30,11 +30,14 @@
   * [Serve Model Endpoint](module/model/user-guide/model\_version\_endpoint.md)
   * [Model Deployment and Serving](module/model/user-guide/model\_deployment\_serving.md)
   * [Batch Prediction](module/model/user-guide/batch\_prediction.md)
+  * [Model Deletion](module/model/user-guide/model_deletion.md)
```

Comment:
Model deletion is currently hidden from the users as it is very slow (only version deletion is possible). Shall we leave it out of these docs?

Reply:
Thanks for the catch! Removed it for now.